resources/views/users/partials/create-user-form.blade.php

```diff
@@ -44,20 +51,45 @@
             <x-form-validation for="admin" />
         </div>
 
-        <!-- User Notes -->
+        <hr>
+
+        <!-- Primary Sector ID -->
         <div>
-            <x-input-label for="notes" :value="__('User Notes')" />
-            <x-text-input id="notes" name="notes" type="text" class="mt-1 block w-full"/>
-            <x-form-validation for="notes" />
+            <x-input-label for="primary_sector_id" :value="__('Primary Sector')" />
+            <x-select-input name="primary_sector_id" id="primary_sector_id" class="block w-64 text-sm">
+                <option class="text-gray-400" value="">-None-</option>
+                @foreach($sectors as $sector)
+                    <option value="{{ $sector->id }}">{{ $sector->name }}</option>
+                @endforeach
+            </x-select-input>
+            <x-form-validation for="primary_sector_id" />
         </div>
 
+        <!-- Primary Department ID -->
+        <div>
+            <x-input-label for="password" :value="__('Primary Department')" />
+            <x-select-input name="primary_dept_id" id="primary_dept_id" class="block w-64 text-sm"> <!-- required> -->
+                <option value="">-None-</option>
+                @foreach($departments as $department)
+                    <option value="{{ $department->id }}">
+                        {{ $department->name }}
+                    </option>
+                @endforeach
+                <!-- Options will be populated by JavaScript based on the selected sector -->
+            </x-select-input>
+            <x-form-validation for="primary_dept_id" />
+        </div>
+
+
+        <hr>
+
         <!-- Password -->
         <div>
             <div style="display: flex; direction: column">
```

Comment:
Remove inline styles, Remember we're using tailwind.
https://tailwindcss.com/docs/flex

Reply:
addressed in 74e62cc125ed9cc905ec3298adb1494f0d33e7f1. I am very aware we are using tailwind, I just forget momentarily and do what is easiest.